docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM alpine:latest as build
```

Comment:
I'd have started from node:16-alpine, but I'm no expert on Docker.

Reply:
Yeah, using alpine as the base oddly makes for a much smaller image (double the size-ish [see https://github.com/remotestorage/armadietto/pull/37#issuecomment-846534169])